src/app/admin/CandidateResponses.tsx

```diff
@@ -56,19 +57,40 @@ const fetchCandidates = async (): Promise<CandidateData[]> => {
 export const CandidateResponses = () => {
     const [modalOpen, setModalOpen] = useState(false);
     const [selectedChallenge, setSelectedChallenge] = useState('');
+    const [searchTerm, setSearchTerm] = useState('');
+    const [errorMessage, setErrorMessage] = useState<string | null>(null);
```

Comment:
Do you need this? Why not just display the error directly?

Reply:
Fixed it